app/Console/Commands/DeleteUser.php

```diff
@@ -39,15 +39,22 @@ public function __construct()
      */
     public function handle()
     {
-        $headers = ['ID','username','realname','email'];
         $user = $this->argument('user');
-        $user_list = User::select('user_id','username','realname','email')->where('username', 'like', '%'.$user.'%')->orWhere('realname','like', '%'.$user.'%')->get()->toArray();
-        $this->table($headers,$user_list);
-
-        $userid = $this->ask('Enter the ID you wish to remove');
-        $remove_user = User::find($userid);
-        if ($this->confirm('Do you wish to remove '.$remove_user->realname.'? [y|N]')) {
-            $remove_user->delete();
+        $user_list = User::select('username')->where('username', 'like', '%'.$user.'%')->orWhere('realname','like', '%'.$user.'%')->get();
```

Comment:
@murrant  I'm searching by realname and username just to make it a little easier to find accounts. Did we want to do this or just by username?